### PR TITLE
Remove redundant GDS SSO initializer

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/epNWudCR/176-write-an-rfc-for-continuous-deployment-and-seek-review

This was made redundant in [1].

[1]: https://github.com/alphagov/gds-sso/pull/241

### CODE REVIEW

File: config/initializers/gds_sso.rb [hunk 1]
```diff
@@ -1,10 +1,3 @@
-GDS::SSO.config do |config|
-  config.user_model   = "User"
-  config.oauth_id     = ENV["OAUTH_ID"]
-  config.oauth_secret = ENV["OAUTH_SECRET"]
-  config.oauth_root_url = Plek.new.external_url_for("signon")
-end
-
 if Rails.env.development?
```
Comment (kevindew) [inline config/initializers/gds_sso.rb:1]: 😱 weird place for a seed :-/

**Review by kevindew — APPROVED** [review]

File: config/initializers/gds_sso.rb (hunk 1 above)
Comment (benthorner) [reply config/initializers/gds_sso.rb:1]: Yeah, it surprised me as well. I thought about it a bit more and decided to add another commit. We still can't remove the initializer entirely, due to the behaviour of the gem.

Comment (benthorner) [reply config/initializers/gds_sso.rb:1]: Could you take another look?

File: config/initializers/gds_sso.rb [hunk 2]
```diff
@@ -1,21 +1,9 @@
...
   #
   # The easiest way to do this is just to override the GDS::SSO test user with a
   # new user we create here.
-  #
-  GDS::SSO.test_user = User.find_or_create_by!(email: "user@test.example").tap do |u|
-    u.name = "Test User"
-    u.permissions = %w[signin manage_short_urls request_short_urls advanced_options receive_notifications]
-    u.save!
-  end
+  GDS::SSO.test_user = User.first
```
Comment (kevindew) [inline config/initializers/gds_sso.rb:8]: I think this is now unnecessary due to this line: https://github.com/alphagov/gds-sso/blob/afdf773ac6478935129c19436b4cf40486169d8f/lib/gds-sso/warden_config.rb#L75

File: config/initializers/gds_sso.rb (hunk 1 above)
Comment (kevindew) [reply config/initializers/gds_sso.rb:1]: Oh nice one Ben - didn't expect you to follow up on this ⭐

File: config/initializers/gds_sso.rb (hunk 2 above)
Comment (benthorner) [reply config/initializers/gds_sso.rb]: Oh yeah, the comment in the file threw me off. Yes, we can delete this.

Comment (benthorner) [reply config/initializers/gds_sso.rb]: Updated 👍

**Review by kevindew — APPROVED** [review]

**Pull request merged**